packages/valory/skills/transaction_settlement_abci/rounds.py

```diff
@@ -340,6 +342,7 @@ class RandomnessTransactionSubmissionRound(CollectSameUntilThresholdRound):
     payload_class = RandomnessPayload
     synchronized_data_class = SynchronizedData
     done_event = Event.DONE
+    none_event = Event.NONE
```

Comment:
The none event cannot be triggered here:

https://github.com/valory-xyz/open-autonomy/blob/5c0b0a81160baf6eb58b710fb0bb9eb10e78bc21/packages/valory/skills/squads_transaction_settlement_abci/payloads.py#L31-L32

Let's do the same as in `SelectKeeperTransactionSubmissionARound`.